### PR TITLE
[DOC] Updating long description

### PR DESCRIPTION
**Pull request by oesteban**
## Changes proposed in this pull request

Update long description field of the package, as it is veery outdated

## Documentation that should be reviewed
Just that paragraph

## Pull-request guidelines:

1. We invite you to list yourself as a *fMRIPrep* contributor, so if your name 
   is not already mentioned, please modify the 
   [``.zenodo.json``](https://github.com/poldracklab/fmriprep/blob/master/.zenodo.json)
   file with your data right above Russ' entry. Example:
   ```
   {
      "name": "Contributor, New FMRIPrep",
      "affiliation": "Department of fMRI prep'ing, Open Science Made-Up University",
      "orcid": "<your id>"
   },
   {
      "name": "Poldrack, Russell A.",
      "affiliation": "Department of Psychology, Stanford University",
      "orcid": "0000-0001-6755-0259"
   },
   ```
   
2. By submitting this request you acknowledge that your contributions are available under the BSD 3-Clause license.

3. Use a descriptive prefix, between brackets for your PR: ``ENH`` (enhancement), ``FIX``, ``TST``, ``DOC``, ``STY``,
   ``REF`` (refactor), ``CI`` (continous integration), ``MAINT`` (maintenance). Example:
   ```
   [ENH] Support for SB-reference in multi-band datasets
   ```
   For works-in-progress, add the ``WIP`` tag in addition to the descriptive prefix. 
   Pull-requests tagged with ``[WIP]`` will not merged in until the tag is removed.

4. Your PR will be reviewed according to the following
   [template](https://github.com/poldracklab/fmriprep/wiki/Reviewing-a-Pull-Request).

5. Documentation is a fundamental aspect to the *glass-box* philosophy that *fMRIPrep* abides by.
   Please understand that the *fMRIPrep* team may (are likely to) request you to improve the documentation
   provided with this PR, within the PR or in future PRs.
   

**Please review and check the following**:

  - [x] I have read the [guidelines for contributions](https://github.com/poldracklab/fmriprep/blob/master/CONTRIBUTING.md).
  - [x] I understand that my contributions will not be merged unless the work is
        finished (i.e. no ``[WIP]`` tag remains in the title of my PR) and tests pass.
  - [x] The proposed code follows the
        [coding style](https://github.com/poldracklab/fmriprep/blob/master/CONTRIBUTING.md#fmriprep-coding-style-guide),
        to the extent I understood them (and I will address any comments raised by the PR's reviewers in this regard).
  - [ ] \(Optional\) I opt-out from being listed in the `.zenodo.json` file.

### CODE REVIEW

File: fmriprep/info.py [hunk 1]
```diff
@@ -24,27 +24,26 @@
...
+integration framework for software-testing, we show that fMRIPrep robustly produces
+high-quality results on a diverse fMRI data collection comprising participants from
+54 different studies in the OpenfMRI repository. We review the distinctive features of
+fMRIPrep in a qualitative comparison to other preprocessing workflows. FMRIPrep achieves
+higher spatial accuracy as it introduces less uncontrolled spatial smoothness than commonly
+used preprocessing tools. FMRIPrep has the potential to transform fMRI research by equipping
+neuroscientists with a high-quality, robust, easy-to-use and transparent preprocessing workflow
+which can help ensure the validity of inference and the interpretability of their results.
+
+[Pre-print https://doi.org/10.1101/306951]"""
```
Comment (effigies) [inline fmriprep/info.py]: I don't think this link is valid RST. You can check with the PyPI pre-check from nipype:

https://github.com/nipy/nipype/blob/master/.circleci/config.yml#L333-L344

If it seems worth adding to our own Circle workflows, it's a 30s job.

Comment (oesteban) [reply fmriprep/info.py]: Will do

**Comment by oesteban** [comment]
This PR also renames `fmriprep/info.py` to `fmriprep/__about__.py` which feels more pythonic.

File: fmriprep/info.py (hunk 1 above)
Comment (oesteban) [reply fmriprep/info.py]: Added

**Comment by oesteban** [comment]
This is ready to merge @effigies, @chrisfilo. In case you want to have a look. Please disregard the WIP bot that seems blocked since the [skip ci] commit.

File: fmriprep/__about__.py [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+reproducible preprocessing for task-based and resting fMRI data.
+FMRIPrep automatically adapts a best-in-breed workflow to the idiosyncrasies of
+virtually any dataset, ensuring high-quality preprocessing with no manual intervention,
+while providing easily interpretable and comprehensive error and output reporting.
+It performs basic preprocessing steps (coregistration, normalization, unwarping, noise
+component extraction, segmentation, skullstripping etc.) providing outputs that can be
+easily submitted to a variety of group level analyses, including task-based or resting-state
+fMRI, graph theory measures, surface or volume-based statistics, etc.
+
+The workflow is based on `Nipype <http://nipype.readthedocs.io>`_ and encompases a large
```
Comment (effigies) [inline fmriprep/__about__.py]: https

File: fmriprep/__about__.py [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+    meaningful errors.
+  * Automate and parallelize processing steps, which provides a significant speed-up from
+    typical linear, manual processing.
+
+FMRIPrep has the potential to transform fMRI research by equipping
+neuroscientists with a high-quality, robust, easy-to-use and transparent preprocessing workflow
+which can help ensure the validity of inference and the interpretability of their results.
+
+[Pre-print doi:`10.1101/306951 <https://doi.org/10.1101/306951>`_]
+[Documentation `fmriprep.org <http://fmriprep.readthedocs.io>`_]
```
Comment (effigies) [inline fmriprep/__about__.py]: https

**Comment by oesteban** [comment]
Basically, one can have a look to the ``fmriprep.egg-info/PKG-INFO`` and manually check it:

```
Metadata-Version: 2.1
Name: fmriprep
Version: 1.1.4+44.g6a3376a1
Summary: FMRIprep is a functional magnetic resonance image pre-processing pipeline that is designed to provide an easily accessible, state-of-the-art interface that is robust to dif
ferences in scan acquisition protocols and that requires minimal user input, while providing easily interpretable and comprehensive error and output reporting.
Home-page: https://github.com/poldracklab/fmriprep
Author: The CRN developers
Author-email: crn.poldracklab@gmail.com
Maintainer: Ross Blair
Maintainer-email: crn.poldracklab@gmail.com
License: 3-clause BSD
Download-URL: https://github.com/poldracklab/fmriprep/archive/1.1.4+44.g6a3376a1.tar.gz
Description: Preprocessing of functional MRI (fMRI) involves numerous steps to clean and standardize
        data before statistical analysis.
        Generally, researchers create ad hoc preprocessing workflows for each new dataset,
        building upon a large inventory of tools available for each step.
        The complexity of these workflows has snowballed with rapid advances in MR data
        acquisition and image processing techniques.
        FMRIPrep is an analysis-agnostic tool that addresses the challenge of robust and
....
```
I'm not sure it is worth the pain for a field that changes every 2 years...

**Comment by oesteban** [comment]
OR
https://pypi.org/project/readme_renderer/

**Pull request merged**